### ISSUE TITLE
process is left running after ctrl-c

### ISSUE THREAD

**Issue by alexeagle** [bazelbuild/bazel-watcher#99]
I'm on the latest npm release. I notice that a single ctrl-c kills the ibazel process now, where it used to require two. And now the process being run is not shut down, so I have to go find and kill it before I can run again.

/cc @gregmagolan 

```
alexeagle@alexeagle:~/Projects/angular-bazel-example$ ./node_modules/.bin/ibazel
iBazel - Version v0.2.0
...
alexeagle@alexeagle:~/Projects/angular-bazel-example$ ps aux | grep devserver
alexeag+ 191626  0.0  0.0  23776  2192 pts/42   S+   09:30   0:00 grep --color=auto devserver
alexeagle@alexeagle:~/Projects/angular-bazel-example$ ./node_modules/.bin/ibazel run src:devserver
State: QUERY
Querying for BUILD files...
Watching: 5 files
Querying for source files...
Watching: 3871 files
State: RUN
Runing src:devserver
Launching with live reload
Launching with notifications
[live reload] listening on :35729
INFO: Analysed target //src:devserver (44 packages loaded).
INFO: Found 1 target...
Target //src:devserver up-to-date:
  dist/bin/src/devserver.MF
  dist/bin/src/scripts_devserver.MF
  dist/bin/src/devserver
INFO: Elapsed time: 3.476s, Critical Path: 1.57s
INFO: Build completed successfully, 1 total action

Starting...Triggering live reload
[live reload] requesting reload: reload
State: WAIT
Serving livereload script from http://localhost:35729/livereload.js?snipver=1
Server listening on http://alexeagle.mtv.corp.google.com:5432/
[live reload] connected
Changed: "src/lib/file.ts". Rebuilding...
State: DEBOUNCE_RUN
State: DEBOUNCE_RUN
State: RUN
Runing src:devserver
Notifying of changes
INFO: Analysed target //src:devserver (0 packages loaded).
INFO: Found 1 target...
Target //src:devserver up-to-date:
  dist/bin/src/devserver.MF
  dist/bin/src/scripts_devserver.MF
  dist/bin/src/devserver
INFO: Elapsed time: 2.491s, Critical Path: 0.22s
INFO: Build completed successfully, 2 total actions
SUCCESS
Triggering live reload
[live reload] requesting reload: reload
State: WAIT
[live reload] websocket: close 1001 (going away)
[live reload] connected
Changed: "src/lib/file.ts". Rebuilding...
State: DEBOUNCE_RUN
State: DEBOUNCE_RUN
State: RUN
Runing src:devserver
Notifying of changes
INFO: Analysed target //src:devserver (0 packages loaded).
INFO: Found 1 target...
Target //src:devserver up-to-date:
  dist/bin/src/devserver.MF
  dist/bin/src/scripts_devserver.MF
  dist/bin/src/devserver
INFO: Elapsed time: 2.285s, Critical Path: 0.20s
INFO: Build completed successfully, 2 total actions
SUCCESS
Triggering live reload
[live reload] requesting reload: reload
State: WAIT
[live reload] websocket: close 1001 (going away)
[live reload] connected
^C
Subprocess killed from getting SIGINT

alexeagle@alexeagle:~/Projects/angular-bazel-example$ 
alexeagle@alexeagle:~/Projects/angular-bazel-example$ ps aux | grep devserver
alexeag+ 192013  0.1  0.0  17976 11324 pts/42   Sl   09:32   0:00 /usr/local/google/home/alexeagle/Projects/angular-bazel-example/node_modules/@bazel/ibazel/bin/linux_amd64/ibazel run src:devserver
alexeag+ 195149  0.0  0.0  23776  2144 pts/42   S+   09:33   0:00 grep --color=auto devserver
```

**Comment by gregmagolan** [comment]
I've seen this as well but it only happens occasionally for me where the process keeps running and taking up port 5432. Haven't been able to determine what causes it. Could just be just random based on a race condition.

**Comment by gregmagolan** [comment]
I did a little investigation and on my machine OSX 10.13 (High Sierra) the process is killed properly with CTRL-C but if I close and terminate the terminal while iBazel is running than it lives on.

**Comment by gregmagolan** [comment]
#102 fixes a shutdown issue with the npm iBazel distribution

**Comment by achew22** [comment]
Fixed by @gregmagolan in 4d886ff. Thanks Greg!

**Comment by appsforartists** [comment]
Can we please push a new release?

Since we recommend that web developers use `npm` to load `ibazel`, this is a pretty big bug (though admittedly, the web dev community on Bazel is still pretty small).
